pom.xml

```diff
@@ -164,9 +164,11 @@
           <target>1.7</target>
           <excludes>
             <exclude>com/datalogics/pdf/samples/extraction/TextCharacterBoxes.java</exclude>
+	    <exclude>com/datalogics/pdf/samples/forms/ExportFormDataToCsv.java</exclude>
```

Comment:
This fails on CI because the ``pom.xml`` changes during the build.

- Don't use tabs in XML, use spaces.
- Run a build or ``sortpom:sort`` before checking in POM changes. Even just building once with ``mvn`` will put all the POMs into canonical form.
- Please correct this and rebase the fix into this commit.